bedrock/settings/base.py

```diff
@@ -847,3 +847,10 @@ def facebook_tab_url_lazy():
 STATSD_HOST = config('STATSD_HOST', default='127.0.0.1')
 STATSD_PORT = config('STATSD_PORT', cast=int, default=8125)
 STATSD_PREFIX = config('STATSD_PREFIX', default='bedrock')
+
+# Thunderbird pre-release versions. Remove this once the new Ship-It library
+# provides the data. (Bug 1150649)
+THUNDERBIRD_VERSIONS = {
```

Comment:
This will almost certainly become out-of-date quickly. I see that there is a mention of `40.0b1` in product-details in the `thunderbird_history_development_releases.json` file, but it would be a little tricky to get it out of there reliably. If we could, we could then infer the alpha version. It's potentially error prone but possibly more reliable than having to remember to update this setting. What do you think?